app/models/profile.py

```diff
@@ -205,10 +205,16 @@ def validate_graph(self, data_graph: Graph) -> bool:
         Raises:
             GraphNotConformError: If not conform, containing the reason why.
         """
-
-        # Check if the graph is empty, as this conforms against the SHACL.
-        if len(data_graph) == 0:
-            raise GraphNotConformError("Empty graph")
+        # An empty graph conforms with the SHACL. Check if the graph has at least
+        # a premis:intellecutalEntity node.
+        if (
```

Comment:
een lijst van assert() (of if) statements per profiel die checken wat shacl niet kan checken lijkt mij wel handig

Reply:
Ja, een soort rule-based check meer lijkt me wel praktisch en ook nodig in de toekomst. For now is deze ene if-statement goed genoeg.

Reply:
check, onderwerp van een toekomstige refactor